CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 ### Fixed
 * Support for regex `mapped-databases` for MANUAL database resolution. See [#147](https://github.com/HotelsDotCom/waggle-dance/issues/147).
 * Avoid NPE when no elements are provided for `mapped-databases` in the configuration. See [#131](https://github.com/HotelsDotCom/waggle-dance/issues/131).
+### Changed
+* Added invocation log messages for `getPartitions` calls.
```

Comment:
is it added or changed ?

Reply:
I change the annotation and the log line is added. I dunno! :)